data/map planets.txt

```diff
@@ -4958,3 +4958,69 @@ planet Zydee
 	landscape land/enceladus_1
 	description `Although this moon initially seems barren and lifeless, there is evidence that the nearby Gegno Scin have recently attempted to set up an outpost here. Several partially-constructed buildings and landing zones are scattered about the surface, but none of them show any signs of activity, likely due to recent warfare with the Gegno Vi. Despite this, a small number of satellites actively orbit Zydee, likely providing important tactical information such as the volume of system traffic.`
 	government Uninhabited
+
+
+# Pirate Stuff
+
+planet Gagarin
+	attributes CCOR factory urban
+	landscape land/city0
+#	description "Gagarin is the oldest-settled planet in the CCOR and the most populated. All the settlements of Gagarin are connected to a planetary transport and infrastructure grid... though the grid hasn't been able to keep pace with demand from a growing population. Although they were designed to be expanded with demand over time, such resource-intensive infrastructure projects became harder and harder to supply the materials for."
```

Comment:
```suggestion
#	description "Gagarin is the oldest-settled planet in the CCOR region and the most populated. All the settlements of Gagarin are connected to a planetary transport and infrastructure grid... though the grid hasn't been able to keep pace with demand from a growing population. Although they were designed to be expanded with demand over time, such resource-intensive infrastructure projects became harder and harder to supply the materials for."
```

Reply:
Would it be the CCOR region, or the region of what used to be the CCOR?